WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -70,4 +72,23 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
++ (id<FBResponsePayload>)handleDismissKeyboardCommand:(FBRouteRequest *)request
+{
+  [request.session.application dismissKeyboard];
```

Comment:
Yes, I tested it and it works merely on iPad (or where **Hide keyboard** button is explicitly present). But it's anyway not our issue.

This was also the reason why I added an error message and keyboard presence verification - to let end users know that the call does not work and they must use custom approaches to hide the keyboard instead.